lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -666,6 +666,19 @@ export class AuthorizationCodeClient extends BaseClient {
             );
         }
 
+        this.performanceClient?.addFields(
+            {
+                useLoginHint: parameterBuilder.hasParameter(
```

Comment:
Would be useful to differentiate between UPN vs. claim if possible

Reply:
good point, done